contracts/cw1-subkeys/src/multitest/tests.rs

```diff
@@ -42,8 +42,11 @@ fn get_contract_version_works() {
         .call(owner)
         .unwrap();
 
-    let version: ContractVersion =
-        query_contract_info(&app.app().wrap(), contract.contract_addr.to_string()).unwrap();
+    let version: ContractVersion = query_contract_info(
+        &(*app).borrow_mut().wrap(),
```

Comment:
Nah, `Deref` doesn't work here as we would like to - we don't want to expose `RefCell`. It was my bad; you were right here. `.app()` was better (and we need to keep it), but because queries are common let's implement the `https://docs.rs/cosmwasm-std/latest/cosmwasm_std/struct.QuerierWrapper.html` directly on our `App`, so we can provide the direct `wrap()` function on `Querier` (and let's rename it to `querier` as the `wrap` name without the context is confusing to me.

Reply:
tracked under #154